bertopic/_bertopic.py

```diff
@@ -1897,9 +1903,40 @@ def get_topic_tree(
 
         max_original_topic = hier_topics.Parent_ID.astype(int).min() - 1
 
+        # Prepare tree labels to print
+        child_left_ids = hier_topics.Child_Left_ID.astype(int)
+        child_right_ids = hier_topics.Child_Right_ID.astype(int)
+
+        # Get the new parent labels generated from `hierarchical_topics`
+        new_left_labels = {int(row["Child_Left_ID"]): row["Child_Left_Name"] for idx, row in hier_topics.iterrows()}
+        new_right_labels = {int(row["Child_Right_ID"]): row["Child_Right_Name"] for idx, row in hier_topics.iterrows()}
+
+        if custom_labels:
+            left_labels = {}
+            if isinstance(custom_labels, str):
+                for topic, kws_info in self.topic_aspects_[custom_labels].items():
+                    label = "_".join([kw[0] for kw in kws_info[:5]])  # displaying top 5 kws
```

Comment:
Note that some topic aspects might not have 5 keywords but actually a single label. In those cases, the `kws_info[0]` is likely to be filled with a label (or even a summary) and each instance in `kws_info[1:]` will be an empty string. As a result, you might get the following label: "artificial intelligence____".